### PR TITLE
TD-26 add 3D Viewer

### PR DESCRIPTION
**Pull request by NadyaG**
* Added mentions of 3D Viewer to all sections.
* Made the query parameters list into a table, with +/- showing for which component each parameter is supported.
* Noted that the events for current model make no sense for 3D Viewer.
* Noted that 3D Viewer would work just as well on a desktop.
* DIDN'T remove the mobile/desktop structure of the guide - should I?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,14 +17,16 @@ In this tutorial, you will be integrating a fitting room that was already config
...
+Virtual Fitting Room could only be comfortably used for virtual try-on [on a smartphone](#mobile-show-the-fitting-room). You probably already have different versions of your website for desktop and mobile devices, but if not, you will need to implement the selection on your side. For [desktop version](#desktop-display-a-qr-code) we recommend displaying a QR code with the link to smoothly guide the user to the virtual try-on page.
+
+The 3D Viewer component may be displayed on both desktop and mobile devices.
 
 ## Mobile: show the fitting room
 
-The essential part of integration is simply to load Virtual Fitting Room into an iframe on the page or just open it in a separate page. Choose the look and user experience you would prefer. There are three main options:
+The essential part of integration is simply to load the Virtual Fitting Room or 3D Viewer component into an iframe on the page or just open it in a separate page. Choose the look and user experience you would prefer. There are three main options:
```
Comment (olgati) [inline README.md:26]: We can keep Virtual Fitting Room here. We use it as a generic name for the solution that can present either virtual tey-on or 3D Viewer

File: README.md [hunk 2]
```diff
@@ -42,19 +44,33 @@ iframe.src = vfrLink
 
 ### Iframe settings
 
-Add query parameters to the link to configure the optional settings:
+Add query parameters to the link to configure the component behavior. Virtual Fitting Room and 3D Viewer parameters differ slightly.
```
Comment (olgati) [inline README.md:47]: Here we need to change Virtual Fitting Room -> Virtual Try-On (see the comment above)

File: README.md [hunk 3]
```diff
@@ -42,19 +44,33 @@ iframe.src = vfrLink
...
 
-* `modelid` — a comma-separated list of model identifiers for the models that should be displayed for try-on.
-* `startwithid` — the identifier of the model that should be loaded first. Note that the order of the models won't change.
-* `locale` — the locale code for the Virtual Fitting Room UI language and metadata. If the specified locale isn't available, the English locale will be loaded instead.
+| Parameter               | Fitting Room      | 3D Viewer                 | Description                                                                                      |
```
Comment (olgati) [inline README.md:49]: The same: Fitting Room -> Virtual Try-On

File: README.md [hunk 4]
```diff
@@ -42,19 +44,33 @@ iframe.src = vfrLink
...
+|-------------------------| :---------------: | :-----------------------: |--------------------------------------------------------------------------------------------------|
+| `modelid`, **required** | **+**<br />a list | **+**<br />a single value | The model identifiers for the models that should be displayed in the component.                                   |
```
Comment (olgati) [inline README.md:51]: modelid is mandatory for a 3D Viewer. We need to add this notice

File: README.md [hunk 5]
```diff
@@ -42,19 +44,33 @@ iframe.src = vfrLink
...
+The following link will show the tutorial when 3D Viewer is loaded and display the interface and page metadata in Italian:
+
+```
+https://bags-demo.ar.wanna.fashion/?modelid=wanna_bag&showonboarding=3d&locale=it
+```
 
 ### Get the current model ID
 
-It should be easy for the user to buy or add to favorites the model they're trying on. You may want to add, for example, an "Add to cart" button to the page footer. To do that, you will need to get in real time the identifier of the model that is being tried on now. Listen to events from the fitting room. Virtual Fitting Room raises the `MODEL_SET` event that contains the ID of the model that is currently loaded. The `data` property of event has following properties:
+**Note:** This section only makes sense for Virtual Fitting Room, as in 3D Viewer you're always loading a single model and know its identifier.
```
Comment (olgati) [inline README.md:71]: Again, Virtual Fitting Room -> Virtual Try-On

File: README.md [hunk 6]
```diff
@@ -85,3 +101,5 @@ Consult also [our demo sample](samples/iframe_mobile.html) which shows the simpl
 ## Desktop: display a QR code
 
 On the product page from which you would like the client to start virtual try-on, display a QR code that the client can scan with their phone and go directly to Virtual Fitting Room on mobile. [Our sample](samples/desktop.html) uses the open-source [QR-Code-generator](https://github.com/nayuki/QR-Code-generator) library for demo purposes. Choose any QR code generating tool that suits you.
+
+For the 3D Viewer component, integrate the iframe in the same way as you would [on mobile](#mobile-show-the-fitting-room). The only difference is be that you set the `viewmode` query parameter to `desktop` instead of `mobile`.
```
Comment (olgati) [inline README.md:105]: Can we add here sample images from the slide https://docs.google.com/presentation/d/1DMPs6F33LOCvE0twhrVQXIcGRTf_4LGKoJl0pvDjG8k/edit#slide=id.g1fcb4a2b870_0_25

**Review by olgati — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (NadyaG) [reply README.md:51]: It says **required** in the first column

Comment (olgati) [reply README.md:51]: It is required only for a 3D Viewer setup. For VTO this parameter is optional.

File: README.md (hunk 6 above)
Comment (NadyaG) [reply README.md]: Done, converting to a regular PR and waiting for the final review.

**Review by vpenyazkov — APPROVED** [review]

**Review by olgati — APPROVED** [review]

**Pull request merged**